File: Inverted Half Pyramid/invertedHalfPyramid.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// Function to print star pattern.
function invertedHalfPyramid(n) {
  for (let i=0; i<n; i++) {
    let pattern = "";
    for (let j=i; j<n; j++) {
      pattern += "*";
    }
    console.log(pattern);
  }
}

// value of n from user and calling invertedHalfPyramid on them.
rl.question('Enter the value of n : ', n => {
  invertedHalfPyramid(Number(n));
  rl.close();
});

File: Inverted Half Pyramid/invertedHalfPyramid.java
import java.util.Scanner;

class invertedHalfPyramid {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter the value of n : ");
    int n = sc.nextInt();

    for (int i=0; i<n; i++) {
      for (int j=i; j<n; j++) {
        System.out.print("*");
      }
      System.out.print("\n");
    }

    sc.close();
  }
}

